providers/route53/route53Provider.go

```diff
@@ -305,3 +385,31 @@ func (r *route53Provider) EnsureDomainExists(domain string) error {
 	return err
 
 }
+
+func (r *route53Provider) waitUntilNameserversUpdate(operationId *string) error {
```

Comment:
If you query the nameservers via api after getting an operation id but before the wait completes, what do you get? If you get the updated nameservers, I'd vote for not bothering to wait. 

In other words, as long as I can run dnscontrol twice in a row and get no corrections on the second run, I'm probably ok with that consistency model.